### ISSUE TITLE
http_cfg_append only handles Hash despite allowing Array

### ISSUE THREAD

**Issue by tux-o-matic** [voxpupuli/puppet-nginx#1173]
The [Data Type](https://github.com/voxpupuli/puppet-nginx/blob/6a4657e7319136e563479b8fab5bf83a2ab6e36d/manifests/init.pp#L86) specified for 'http_cfg_append only' allows Hash and Array but the [ERB template](https://github.com/voxpupuli/puppet-nginx/blob/master/templates/conf.d/nginx.conf.erb#L188) only handles Hash containing Array.

**Comment by skeldoy** [comment]
Yes. This is so cool if you have two identical lines (like two closing brackets on different lines) and you have to add some comment to the line to make it unique in order for the append to work.. Very annoying.
